src/shim/util/wrapper.ts

```diff
@@ -0,0 +1,18 @@
+import global from '../global';
+import has from '../../has/has';
+
+export default function wrapper(nameOnGlobal: string, constructor = false): any {
+	if (has('test')) {
+		if (constructor) {
+			return function(...args: any[]) {
+				return new global[nameOnGlobal](...args);
+			};
+		} else {
+			return function(...args: any[]) {
+				return global[nameOnGlobal](...args);
+			};
+		}
+	}
+
+	return global[nameOnGlobal].bind(global);
```

Comment:
The only reason I put this here is because `fetch` was already doing this. I could just put another parameter on `wrapper` to control this and make sure it only happens for `fetch`, or remote that entirely if it wasn't necessary in the first place.

Reply:
@maier49 cool, perhaps it wasn't needed in the first place?

Reply:
The bind on fetch was added here: https://github.com/dojo/framework/pull/192

Reply:
@matt-gadd I can't think of or find any reason why it would be. I've gone ahead and removed it.

Reply:
@maier49 did you see the original issue as to why fetch was bound?

Reply:
@agubler Nope, sorry, missed that. I will add another flag to the wrapper then.

Reply:
Fixed